Tasks/PublishSymbols/Publish-Symbols.ps1

```diff
@@ -165,7 +165,13 @@ function Update-SymbolClient([string]$symbolServiceUri, [string]$symbolClientLoc
     # Check latest package version.
     $availableVersion = Get-SymbolClientVersion $symbolServiceUri
 
-    $symbolPathRoot = Join-Path $env:APPDATA "VSOSymbolClient"
+    $agent = Get-VstsTaskVariable -Name 'agent.version'
+    if (!$agent -or (([version]'2.115.0').CompareTo([version]$agent) -ge 1)) {
+        $symbolPathRoot = Join-Path $env:APPDATA "VSOSymbolClient"        
+    }
+    else {
+        $symbolPathRoot = Join-Path $env:AGENT_TOOLSDIRECTORY "VSOSymbolClient"
```

Comment:
>$env:AGENT_TOOLSDIRECTORY [](start = 36, length = 25)

Considering that you are taking dependency on Get-VstsTaskVariable function here, 
Please use use Get-VstsTaskVariable here too, to get the agent tools directory